programs/cleos/httpc.cpp

```diff
@@ -258,6 +258,8 @@ namespace eosio { namespace client { namespace http {
       response_result = fc::json::from_string(re);
    } catch(...) {
       // re reported below if print_response requested
+      std::cerr << "Error happens, below is the message received from server:" << std::endl;
```

Comment:
I don't think this cerr is needed. Also you should update the comment.

Reply:
That is for let user know error happened.   only print response don't means an error happened.

Reply:
It is not necessarily an error, right? This only means it is not json.

Reply:
Ok. remove that line.

Reply:
The problem of this function do_http_call is layer chaos, a function in http layer try to handle higher layer thing, that caused you don't even know a exception throw is an error or not.